lnwallet/btcwallet/btcwallet.go

```diff
@@ -997,7 +997,7 @@ func (b *BtcWallet) SendOutputs(outputs []*wire.TxOut,
 
 	return b.wallet.SendOutputs(
 		outputs, nil, defaultAccount, minConfs, feeSatPerKB,
-		b.cfg.CoinSelectionStrategy, label,
+		b.cfg.CoinSelectionStrategy, label, selectedUtxos...,
```

Comment:
here we need to pass in a new strategy if utxos are selected, something that indicates manual selection

Reply:
There is a new PR on lnd to add strategy as an arg to functions like this but for now I just skipped calling `ArrangeCoins` if there is a selected utxo in the btcd side.